examples/eks/prometheus-pipeline/prometheus-sample-app.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# create a namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aoc-prometheus-pipeline-demo
+  labels:
+    name: prometheus-sample-app
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus-sample-app
+  namespace: aoc-prometheus-pipeline-demo
+  labels:
+    app: prometheus-sample-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prometheus-sample-app
+  template:
+    metadata:
+      labels:
+        app: prometheus-sample-app
+    spec:
+      containers:
+      - name: prometheus-sample-app
+        image: "" # change to your local image
```

Comment:
can't it be the sample-app we are going to push?

Reply:
We actually realized we don't have enough time to push the sample app. We'll have to build and push it locally.